jigsawpy/__init__.py

```diff
@@ -50,45 +50,42 @@
 ------------------------------------------------------------
  """
 
-from .msh_t import jigsaw_msh_t
-from .jig_t import jigsaw_jig_t
+from jigsawpy.msh_t import jigsaw_msh_t
+from jigsawpy.jig_t import jigsaw_jig_t
 
-from .def_l import *
+from jigsawpy.def_l import *
 
-from .loadmsh import loadmsh
-from .savemsh import savemsh
-from .loadjig import loadjig
-from .savejig import savejig
+from jigsawpy.loadmsh import loadmsh
+from jigsawpy.savemsh import savemsh
+from jigsawpy.loadjig import loadjig
+from jigsawpy.savejig import savejig
 
-from .certify import certify
+from jigsawpy.certify import certify
+from jigsawpy import jigsaw, libsaw
 
 class cmd :
 #--------------------------------- expose cmd-line interface
     def jigsaw(opts,mesh=None) :
-        from  .jigsaw import jigsaw        
         
-        return jigsaw(opts,mesh)
+        return jigsaw.jigsaw(opts,mesh)
```

Comment:
These kinds of local imports are considered bad practice as well.  There's a performance hit each time the function gets called.  More important in this case, it's confusing that a different `jigsaw` and `tripod` function gets called from different places.  I'd clear this up by doing the import at the top but one level up (`jigsawpy.jigsaw` instead of `jigsawpy.jigsaw.jigsaw`).  This way, it's clear which function is getting called and no local import is required.